bin/i18n/i18n_metrics.rb

```diff
@@ -0,0 +1,44 @@
+require 'cdo/aws/metrics'
+require 'aws-sdk-ec2'
+require 'net/http'
+require 'socket'
+
+module I18n
+  module Metrics
+    class << self
+      attr_accessor :machine_id
+    end
+
+    def report_runtime(addtl_dimensions, value)
```

Comment:
```suggestion
    def self.report_runtime(addtl_dimensions, value)
```

Reply:
Does self mean anything if these methods live in a module, not a class?

Reply:
It becomes defined on the module’s [eigenclass](https://medium.com/@ethan.reid.roberts/rubys-anonymous-eigenclass-putting-the-ei-in-team-ebc1e8f8d668).

If you want to call it from the module as `18n::Metrics.report_runtime`, you have 3 options for defining the method: https://www.mendelowski.com/docs/ruby/self-vs-extend-self-vs-module-function/

I would recommend using the `self` definition for consistency on the project.